content/messaging/agent-chat-api/v3.4/rtm-reference/index.mdx

```diff
@@ -4593,6 +4594,79 @@ Informs that an Agent was disconnected. The payload contains the reason of Custo
 Also, `misdirected_connection` returns the correct `region` value in the optional `data` object.
 With this piece of information, the client is able to figure out where it should be connected.
 
+## Configuration
+
+### `group_created`
+
+Informs that a group has been created on the license.
+
+<CodeResponse title={'Sample push payload'}>
+
+```json
+{
+  "id": 42,
+	"name": "Sales",
+	"language_code": "en",
+	"agent_priorities": {
+    "agent@example.com": "normal",
+    "other_agent@example.com": "first"
+  }
+}
+```
+
+</CodeResponse>
+
+#### Specifics
+
+|                        |                                                                                 |
+| ---------------------- | ------------------------------------------------------------------------------- |
+| **Action**             | [`create_group`](/management/configuration-api/#create_group)                   |
+| **Webhook equivalent** | [`group_created`](/management/configuration-api/#group_created)                 |
+
+
+### `group_deleted`
```

Comment:
Any reason the order is different in the RTM version?

Reply:
Different than where?

Reply:
The order of events in non-RTM was: created, updated, deleted, while in RTM it was: created, deleted, updated. Łukasz has now fixed that.